browser/mocha/stub.js

```diff
@@ -25,20 +33,17 @@ extendInterfaces('stub', function(context, teardown) {
     var proto = document.createElement(tagName).constructor.prototype;
 
     // For all keys in the implementation to stub with..
-    var keys = Object.keys(implementation);
-    keys.forEach(function(key) {
+    var stubs = Object.keys(implementation).forEach(function(key) {
```

Comment:
Sorry was suppose to change this to map instead of forEach, which is why I changed the behavior to return the stub object.